testdata/p4_16_samples/list4.p4

```diff
@@ -0,0 +1,25 @@
+#include <core.p4>
+
+typedef int EntryPriority_t;
+
+extern E<K, V> {
+    E(list<tuple<EntryPriority_t, tuple<K, K>, V>> data);
```

Comment:
Considering alternatives, from Vlad: what about this?
E(tuple<EntryPriority_t, tuple<K, K>, V>{} data);

More generally, the expression `T{}` instead of `list<T>`.

Reply:
This looks like an initializer rather than a type.
I find it confusing as a user.

Reply:
Scratch space to aid some discussion during 2022-Nov-07 LDWG meeting:

Alternate syntax of expression proposed by Vlad would be this, I think: `(tuple<EntryPriority_t, tuple<K, K>, V>{}) {val1, val2, val3}`

If we simplify the example to involve list<bit<5>> instead of the more complex element type in the example above for the below, would the compiler be able to infer the type of {val1, val2, val3} in an instantiation like the one below?

`E<K, V>( {2, 7, 10} } myEinst1;`

AI Mihai: Check whether current p4c implementation allows variables with the name `list` even with these changes, because the compiler can infer it from context?

Andrew Pinski: C23 is reversing an earlier decision not to add new keywords, and saying "Yes, we will add new keywords".